ccdproc/image_collection.py

```diff
@@ -271,7 +284,13 @@ def refresh(self):
         """
         keywords = '*' if self._all_keywords else self.keywords
         # Re-load list of files
-        self._files = self._fits_files_in_directory()
+        if self._filenames:
```

Comment:
@MSeifert04 I added a helper method called ``_get_files()``, do you like this name? xD